library/network/src/lib/network/susefirewall2.rb

```diff
@@ -1627,7 +1627,7 @@ def ActivateConfiguration
       # starting firewall during second stage can cause deadlock in systemd - bnc#798620
       # Moreover, it is not needed. Firewall gets started via dependency on multi-user.target
       # when second stage is over.
-      if Mode.installation
+      if Mode.installation && !Mode.autoinst
```

Comment:
Although merged, the complete class should be removed in short as it should not used anymore